pkg/modules/generators/app_configurations_generator.go

```diff
@@ -134,7 +134,7 @@ func (g *appConfigurationGenerator) Generate(spec *v1.Spec) error {
 
 	// patch workload with resource patchers
 	for _, p := range patchers {
-		if err = patchWorkload(&wl, &p); err != nil {
```

Comment:
Is the interface defined as multi-resource and patcher a better implementation?

Reply:
We will provide a new function to support patching other resources  in the next version

Reply:
ok